diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f0c0f5a96287d667f5f5011bec3801589afac30f..c5ef1f8f568ce62f98c30a184237b60970fe19ce 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2257,6 +2257,7 @@ jumbo_static_library("browser") {
     "//ui/strings",
     "//ui/surface",
     "//ui/web_dialogs",
+    "//brave/browser:browser_process",
   ]
 
   if (is_win) {
