diff --git a/content/browser/frame_host/render_frame_message_filter.cc b/content/browser/frame_host/render_frame_message_filter.cc
index c194f7a03838f72c38bdb57b43e687d1bc1bad27..71d3189b3d5ad1ae70a23113c8969b88e0398bb2 100644
--- a/content/browser/frame_host/render_frame_message_filter.cc
+++ b/content/browser/frame_host/render_frame_message_filter.cc
@@ -18,6 +18,7 @@
 #include "base/syslog_logging.h"
 #include "base/task/post_task.h"
 #include "base/unguessable_token.h"
+#include "brave/components/brave_session/browser/brave_session_token.h"
 #include "build/build_config.h"
 #include "content/browser/bad_message.h"
 #include "content/browser/blob_storage/chrome_blob_storage_context.h"
