diff --git a/third_party/blink/renderer/modules/webaudio/analyser_node.cc b/third_party/blink/renderer/modules/webaudio/analyser_node.cc
index 4df218da56b5f4db4bf40d5ca19c1ab24d9e50fd..02d4fedeae3452b105c872158aac88654d707d79 100644
--- a/third_party/blink/renderer/modules/webaudio/analyser_node.cc
+++ b/third_party/blink/renderer/modules/webaudio/analyser_node.cc
@@ -192,6 +192,7 @@ double AnalyserHandler::TailTime() const {
 AnalyserNode::AnalyserNode(BaseAudioContext& context)
     : AudioBasicInspectorNode(context) {
   SetHandler(AnalyserHandler::Create(*this, context.sampleRate()));
+  BRAVE_ANALYSERNODE_CONSTRUCTOR
 }
 
 AnalyserNode* AnalyserNode::Create(BaseAudioContext& context,
