diff --git a/third_party/blink/renderer/modules/webaudio/analyser_node.h b/third_party/blink/renderer/modules/webaudio/analyser_node.h
index b859ca7b01431271372970b2b790476883b27a75..371591122552fbc8f9d5b421015228a726acbd53 100644
--- a/third_party/blink/renderer/modules/webaudio/analyser_node.h
+++ b/third_party/blink/renderer/modules/webaudio/analyser_node.h
@@ -85,6 +85,7 @@ class AnalyserHandler final : public AudioBasicInspectorHandler {
   bool RequiresTailProcessing() const final;
   double TailTime() const final;
 
+  BRAVE_ANALYSER_NODE_H
  private:
   AnalyserHandler(AudioNode&, float sample_rate);
   bool PropagatesSilence() const override {
